cmd/skopeo/copy.go

```diff
@@ -54,5 +84,15 @@ var copyCmd = cli.Command{
 			Name:  "sign-by",
 			Usage: "Sign the image using a GPG key with the specified `FINGERPRINT`",
 		},
+		cli.StringFlag{
+			Name:  "src-creds, screds",
+			Value: "",
+			Usage: "Use `USERNAME:PASSWORD` for accessing the source registry",
```

Comment:
Could this use `USERNAME[:PASSWORD]`, i.e. does that work with the template-variable markers? (If so, please fix everywhere)